web/src/app/modules/views/main/authentication/modules/login/components/login.component.css

```diff
@@ -1,7 +1,12 @@
 .not-allowed {
     cursor: not-allowed;
 }
+.bgimg
```

Comment:
Please create 1 pull request per feature (revert this change and open a new pull request).

Reply:
We have placed a new pull request titled "Feature-7 (Group-3)Background image for login page ".
Thank you.